contracts/MeetupGroup.sol

```diff
@@ -0,0 +1,125 @@
+    event JoinGroup(address groupId, address userId, uint memberCount);
+    event LeaveGroup(address groupId, address userId, uint memberCount);
+
+
+    /* Public functions */
+    /**
+     * @dev Constructor.
+     * @param _owner Address of the user that creates the group from factory contract
+     * @param _name The name of the meetup group
+     * @param _geohash Geohash of the meetup group
+     * @param _category Category the meetup group belongs to
+     * @param _description Desciption of the meetup group
+     * @param _logo IPFS link
+     
+     */
+    constructor (
+        address _owner,
+        string _name,
+        bytes8 _geohash,
+        string _category,
+        string _description,
+        bytes _logo
+    ) public {
+        require(_owner != 0x0, "Owner needs to be a valid address");
+        owner = _owner;
+
+        if (bytes(_name).length != 0){
+            name = _name;
+        } else {
+            name = "Test";
+        }
+
+        if (_geohash.length != 0){
+            geohash = _geohash;
+        } else {
+            geohash = 0x0;
+        }
+
+        if (bytes(_category).length != 0){
+            category = _category;
+        } else {
+            category = "Tech";
+        }
+
+        if (bytes(_description).length != 0){
+            description = _description;
+        } else {
+            description = "Test description";
+        }
+
+        if (_logo.length != 0){
+            logo = _logo;
+        } else {
+            logo = "0x0";
+        } 
+    }
+
+    /**
+     * @dev Creates an event = a Conference smart contract
+     * @param _name The name of the event
+     * @param _deposit The amount each participant deposits. The default is set to 0.02 Ether. The amount cannot be changed once deployed.
+     * @param _limitOfParticipants The number of participant. The default is set to 20. The number can be changed by the owner of the event.
+     * @param _coolingPeriod The period participants should withdraw their deposit after the event ends. After the cooling period, the event owner can claim the remining deposits.
+     * @param _description Desciption of the event
+     * @param _date Timestamp of the event
+     * @param _geohash Geohash of the event
+     */
+    function createEvent (
+        string _name,
+        uint256 _deposit,
+        uint _limitOfParticipants,
+        uint _coolingPeriod,
+        string _description,
+        uint _date,
+        bytes8 _geohash
+    ) public onlyOwner {
+        address conference = new Conference(
+            msg.sender,
+            _name,
+            _deposit,
+            _limitOfParticipants,
+            _coolingPeriod,
+            _description,
+            _date,
+            _geohash
+        );
+        events.push(conference);
+        emit CreateEvent(conference, this, _name, _description, _date, msg.sender);
```

Comment:
We should emit the location stuff. Also the deposit required and the max attendees. Also should we emit the current amount of attendees just in initialize it in the store? Will this be one?